addons/account/wizard/account_move_send_wizard.py

```diff
@@ -124,16 +138,15 @@ def _compute_sending_method_checkboxes(self):
         """ Select one applicable sending method given the following priority
         1. preferred method set on partner,
         2. email,
-        3. manual.
         """
         methods = self.env['ir.model.fields'].get_field_selection('res.partner', 'invoice_sending_method')
         for wizard in self:
-            preferred_method = self._get_default_sending_method(wizard.move_id)
-            need_fallback = not self._is_applicable_to_move(preferred_method, wizard.move_id)
-            fallback_method = need_fallback and ('email' if self._is_applicable_to_move('email', wizard.move_id) else 'manual')
+            preferred_methods = self._get_default_sending_method(wizard.move_id)
+            need_fallback = not any([self._is_applicable_to_move(preferred_method, wizard.move_id) for preferred_method in preferred_methods])
+            fallback_method = need_fallback and ('email' if self._is_applicable_to_move('email', wizard.move_id) else False)
             wizard.sending_method_checkboxes = {
                 method_key: {
-                    'checked': method_key == preferred_method if not need_fallback else method_key == fallback_method,
+                    'checked': method_key in preferred_methods if not need_fallback else method_key == fallback_method,
```

Comment:
Not correct.
I think you need something like (not tested):
```python
applicable_pref_methods = {preferred_method for preferred_method in self._get_default_sending_method(wizard.move_id) if self._is_applicable_to_move(preferred_method, wizard.move_id)}
```
And remove the fallback logic. Since if we remove "manual", it's not needed, it's perfectly valid to have a wizard with no checkboxes checked.